frontend/lib/src/AppNode.ts

```diff
@@ -197,6 +197,7 @@ export class ElementNode implements AppNode {
       datasets: modifiedDatasets,
       useContainerWidth: proto.useContainerWidth,
       vegaLiteTheme: proto.theme,
+      needsAddRows: proto.needsAddRows,
```

Comment:
Is the proto field here even required? Couldn't we just always set it to false here since the proto change seems to be never set to something else than false?